stglib/core/utils.py

```diff
@@ -1398,3 +1398,24 @@ def _todict(matobj):
         else:
             dic[strg] = elem
     return dic
+
+
+def create_water_level_var(ds):
```

Comment:
What if we don't have z in meters NAVD88?

Reply:
I added an if statement to check if z is in NAVD88 first and if not, to print a message that the water_level variable cannot be created because it is relative to NAVD88.